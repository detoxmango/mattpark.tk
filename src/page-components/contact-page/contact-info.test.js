import React from 'react'
import ShallowRenderer from 'react-test-renderer/shallow'

import ContactInfo from './contact-info'

describe('ContactInfo', () => {
    const renderer = ShallowRenderer.createRenderer()
    let tree

    beforeEach(() => {
        renderer.render(<ContactInfo />)
        tree = renderer.getRenderOutput()
    })

    it('renders correctly', () => {
        expect(tree).toMatchSnapshot()
    })

    it('passes', () => {
        expect(true).toBe(true)
    })
})
