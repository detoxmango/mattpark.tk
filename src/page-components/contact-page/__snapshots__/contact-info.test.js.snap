// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContactInfo renders correctly 1`] = `
<div
  className="makeStyles-root-1"
>
  <WithStyles(ForwardRef(Typography))
    className="makeStyles-myName-2"
    variant="h1"
  >
    Matt Park
  </WithStyles(ForwardRef(Typography))>
  <div
    className="makeStyles-myInfo-3"
  >
    <div
      className="makeStyles-contactInfo-4"
    >
      <WithStyles(ForwardRef(Link))
        className="makeStyles-locationLink-6"
        href="https://goo.gl/maps/qm8HMXfbXSABazKRA"
        rel="noopener noreferrer"
        target="_blank"
      >
        <GoLocation
          className="makeStyles-hoverIcon-5"
        />
        <MdLocationCity />
        <span>
          New York, New York
          <br />
          United States
        </span>
      </WithStyles(ForwardRef(Link))>
      <WithStyles(ForwardRef(Link))
        className="makeStyles-emailLink-8"
        href="mailto:mattparksolns@gmail.com"
        target="_top"
      >
        <RiMailSendLine
          className="makeStyles-hoverIcon-5"
        />
        <RiMailLine />
        <span>
          mattparksolutions@gmail.com
        </span>
      </WithStyles(ForwardRef(Link))>
      <WithStyles(ForwardRef(Link))
        className="makeStyles-phoneLink-7"
        href="tel:+1201-591-3323"
        target="_top"
      >
        <FiPhoneCall
          className="makeStyles-hoverIcon-5"
        />
        <FiPhone />
        <span>
          (201) 591 - 3323
        </span>
      </WithStyles(ForwardRef(Link))>
    </div>
    <Socials
      className="makeStyles-socials-9"
    />
  </div>
</div>
`;
